add_book.php

```diff
@@ -0,0 +1,34 @@
+<?php
+include 'database.php';
```

Comment:
Observations: The HTML and PHP code are mixed, which can make the codebase harder to maintain and read.

Recommendations:
- **Separate Concerns**: Use a templating engine like Twig or Blade to separate PHP logic from HTML presentation. This will improve maintainability and readability.

Example:
- Create a template file `add_book.html.twig`:
```html
<h1>Add a New Book</h1>
<form method="POST">
    <label>Title:</label>
    <input type="text" name="title"><br>
    <label>Author:</label>
    <input type="text" name="author"><br>
    <label>Year:</label>
    <input type="number" name="year"><br>
    <button type="submit">Add Book</button>
</form>
```
- Render the template in `add_book.php`:
```php
$loader = new \Twig\Loader\FilesystemLoader('/path/to/templates');
$twig = new \Twig\Environment($loader);

echo $twig->render('add_book.html.twig');
```